### PR TITLE
Fix/cicd deploy error

### PR DESCRIPTION
**Pull request by SR0725**

### CODE REVIEW

File: apps/frontend_old/src/modules/ui/Navbar.tsx [hunk 1]
```diff
@@ -9,14 +9,14 @@ import {
...
-  PowerIcon,
-} from "@heroicons/react/24/outline";
+// import {
+//   UserCircleIcon,
+//   ChevronDownIcon,
+//   Cog6ToothIcon,
+//   InboxArrowDownIcon,
+//   LifebuoyIcon,
+//   PowerIcon,
+// } from "@heroicons/react/24/outline";
```
Comment (KatoToyoki) [inline apps/frontend_old/src/modules/ui/Navbar.tsx:19]: why don't you just delete them?

File: apps/frontend_old/src/modules/ui/Navbar.tsx [hunk 2]
```diff
@@ -27,23 +27,23 @@ import classNames from "classnames";
 const profileMenuItems = [
   {
     label: "My Profile",
-    icon: UserCircleIcon,
+    icon: 'UserCircleIcon',
```
Comment (KatoToyoki) [inline apps/frontend_old/src/modules/ui/Navbar.tsx:30]: what will it affect

File: apps/web/src/components/gameObjects/artwork/index.tsx
Comment (KatoToyoki) [inline apps/web/src/components/gameObjects/artwork/index.tsx:1]: it just looks like the former one, are there any changes?

File: apps/frontend_old/src/modules/ui/Navbar.tsx (hunk 2 above)
Comment (SR0725) [reply apps/frontend_old/src/modules/ui/Navbar.tsx:30]: frontend_old 之後會被移除掉
可以不用太在意這的檔案裡頭的變更

File: apps/frontend_old/src/modules/ui/Navbar.tsx (hunk 1 above)
Comment (SR0725) [reply apps/frontend_old/src/modules/ui/Navbar.tsx:19]: frontend_old 之後會被移除掉
可以不用太在意這的檔案裡頭的變更

Comment (KatoToyoki) [reply apps/frontend_old/src/modules/ui/Navbar.tsx:19]: the `removed` you mean is that the files will be put in different places or the files will just all be deleted?

File: apps/web/src/components/gameObjects/artwork/index.tsx
Comment (night2021-2021) [reply apps/web/src/components/gameObjects/artwork/index.tsx:1]: Owing to the different EOL character between windows and Linux ,they look same.

Comment (KatoToyoki) [reply apps/web/src/components/gameObjects/artwork/index.tsx:1]: okay i get it

**Review by KatoToyoki — APPROVED** [review]

**Review by night2021-2021 — APPROVED** [review]
LGTM

**Pull request merged**